spec/models/events/steps/personalised_updates_spec.rb

```diff
@@ -0,0 +1,68 @@
+require "rails_helper"
+
+describe Events::Steps::PersonalisedUpdates do
+  include_context "wizard step"
+  include_context "stub types api"
+
+  it_behaves_like "a wizard step"
+
+  context "attributes" do
+    it { is_expected.to respond_to :address_postcode }
+    it { is_expected.to respond_to :degree_status_id }
+    it { is_expected.to respond_to :consideration_journey_stage_id }
+    it { is_expected.to respond_to :preferred_teaching_subject_id }
+  end
+
+  context "validations" do
+    it { is_expected.to allow_value("TE571NG").for :address_postcode }
```

Comment:
These may be easier to read/more concise if we grouped some of them into single blocks:

```
it { is_expected.to allow_values("TE571NG", "TE57 1NG", " TE57 1NG ", "").for :address_postcode }
... etc ...
```